.circleci/config.yml

```diff
@@ -314,8 +314,9 @@ commands:
             CHROME_FLAGS_HEADLESS: "--headless=new --remote-debugging-port=1234"
             CHROME_FLAGS_WASM: "--enable-experimental-webassembly-features --js-flags=\"--experimental-wasm-stack-switching --experimental-wasm-type-reflection\""
             CHROME_FLAGS_NOCACHE: "--disk-cache-dir=/dev/null --disk-cache-size=1 --media-cache-size=1 --disable-application-cache --incognito"
+            CHROME_FLAGS_AUDIO: "--autoplay-policy=no-user-gesture-required --use-fake-device-for-media-stream"
           command: |
-            export EMTEST_BROWSER="/usr/bin/google-chrome $CHROME_FLAGS_BASE $CHROME_FLAGS_HEADLESS $CHROME_FLAGS_WASM $CHROME_FLAGS_NOCACHE"
+            export EMTEST_BROWSER="/usr/bin/google-chrome $CHROME_FLAGS_BASE $CHROME_FLAGS_HEADLESS $CHROME_FLAGS_WASM $CHROME_FLAGS_NOCACHE $CHROME_FLAGS_AUDIO"
```

Comment:
If we make this change should we also remove the `EMTEST_LACKS_SOUND_HARDWARE` setting above?

I'm kind of inclined to make this change as its one separate PR, e.g. `[CI] Enable audio testing on chrome`?

Reply:
I didn't have much time today so it was mostly experimenting and coming back to it, but:

- I'll remove the`EMTEST_LACKS_SOUND_HARDWARE`
- I'll rename to match its new purpose (I wasn't sure what I'd achieve when starting)

Is it fine to always enable the dummy sound hardware and bypass the media interaction? I can't think what this would break (and where real sound hardware is present its used as a preference, at least from my manual testing).

Reply:
I imagine enabling dummy sounds hardware and bypass the interaction check should be fine for all out tests..

Reply:
I limited removing `EMTEST_LACKS_SOUND_HARDWARE` to Chrome (FF I can't get to work, documented in the comments).